docker/jenkins/notarize-release.sh

```diff
@@ -45,91 +45,20 @@ fi
 
 # Submit the notarization request to Apple
 echo "Submitting notarization request using account $APPLE_ID_USR..."
-XCRUN_RESULT="$(mktemp)"
-xcrun altool --notarize-app \
-    --primary-bundle-id "org.rstudio.RStudio" \
-    --username $APPLE_ID_USR \
-    --password "@env:APPLE_ID_PSW" \
-    --file $1 \
-    --output-format xml \
-    -itc_provider RStudioInc > $XCRUN_RESULT
+xcrun notarytool submit --wait \
+    --apple-id $APPLE_ID_USR \
+    --password $APPLE_ID_PSW \
+    --team-id FYF2F5GFX4 \
```

Comment:
Do we document somewhere where this ID comes from? (Or could we add a comment?)

Reply:
Yeah, I can add that in. It's from Jonathan's Apple ID account but we'll eventually have a common team account (just for RStudio) that a couple of people on our team will be able to access.